packages/utilities/src/utilities/scrollIntoView/scrollIntoView.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright IBM Corp. 2016, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import '@carbon/ibmdotcom-styles/scss/internal/scroll-into-view/_scroll-into-view.scss';
```

Comment:
Wondering if we should import our users use this Sass file by themselves, instead of putting the styles in the utilities package, which means:

```suggestion
```